packages/shared/src/components/buttons/ButtonV2.tsx

```diff
@@ -109,7 +109,7 @@ function ButtonComponent<TagName extends AllowedTags>(
       {loading && (
         <Loader
           data-testid="buttonLoader"
-          className="absolute inset-0 m-auto !visible"
+          className="absolute m-auto !visible"
```

Comment:
Why do we actually need `!visible` here?
I don't see a hidden class on it.

Reply:
Checked, doesn't seem to be needed